nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -27,30 +27,40 @@
 import com.google.errorprone.VisitorState;
 import com.sun.source.tree.ExpressionTree;
 import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.code.Type;
 import com.uber.nullaway.NullAway;
 import java.io.DataInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.LinkedHashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.net.JarURLConnection;
+import java.net.URLConnection;
+import java.util.*;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import javax.lang.model.element.Modifier;
+import javax.lang.model.type.TypeKind;
 
 /** This handler loads inferred nullability model from stubs for methods in unannotated packages. */
 public class InferredJARModelsHandler extends BaseNoOpHandler {
+  private static boolean DEBUG = false;
+  private static boolean VERBOSE = false;
```

Comment:
@msridhar I am a bit concerned about having this file-specific logging code. But, on the other hand, it is pretty handy for debugging JarInfer. Should we just land it with these `LOG` calls and then decide whether to either remove them or replace them with some sort of NullAway-wide logging infra?

Reply:
We don't have a general logging story right now, mostly because it's a pain to log from within javac.  I'm ok with keeping this for now and trying to replace it late